transcom/mymove
# B-18391-INT

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18391)

## Summary

Customers with prime counseled PPMs won't be able to have their advance approved in milmove like a normal customer would. It was determined that we should update the instructions a customer receives after they've submitted a PPM with an advance request. To accomplish this I updated orders.go to return the ProvidesServiceCounseling boolean, and on the front end, we conditionally render new instructions.

### How to test

1. Create a new move that will be prime counseled (for example, use NS Mayport 32228 as your current duty location)
2. Create a PPM and request an advance
3. Your instructions should look like this
![image](https://github.com/transcom/mymove/assets/147535220/2cfa967f-25ff-4731-9fa6-1d9edc98ad59)
